### PR TITLE
enh: Add ender chest preview

### LINKED ISSUE #16: Enderchest preview
add enderchest preview to shulker preview maybe saved in a drop packet

### PR DESCRIPTION
**Pull request by scorbett123**
Add ender chest preview to Container preview (renamed from Shulker Preview).

Closes #16

### CODE REVIEW

File: src/main/java/com/lambda/client/mixin/client/gui/MixinGuiChest.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+@Mixin(GuiChest.class)
+public class MixinGuiChest {
+
+    @Final @Shadow private IInventory lowerChestInventory;
+
+    @Inject(method = "drawScreen", at = @At("RETURN"))
+    public void drawScreen(int mouseX, int mouseY, float partialTicks, CallbackInfo ci) {
+        if (lowerChestInventory.getName().equals("Ender Chest")) {
+            System.out.println("They are here " + lowerChestInventory.getStackInSlot(0));
```
Comment (ToxicAven) [inline src/main/java/com/lambda/client/mixin/client/gui/MixinGuiChest.java]: Remove Logging?

**Review by ToxicAven — CHANGES_REQUESTED** [review]
Looks pretty good!

File: src/main/java/com/lambda/client/mixin/client/gui/MixinGuiChest.java (hunk 1 above)
Comment (scorbett123) [reply src/main/java/com/lambda/client/mixin/client/gui/MixinGuiChest.java]: Oh, yea, sorry, forgot about that being there.

**Review by ToxicAven — APPROVED** [review]
👍

**Review by Natan515 — APPROVED** [review]

**Review by czho — APPROVED** [review]

**Review by Avanatiker — APPROVED** [review]

**Pull request merged**